lib/doctor/docker.rb

```diff
@@ -0,0 +1,50 @@
+module Doctor
```

Comment:
I think you could use https://github.com/alphagov/govuk-docker/blob/master/lib/doctor/checkup.rb instead of needing this class. Something like `Doctor::Checkup.new(service_name: 'docker', checkups: [], messages: []).installed?` (perhaps worth making `checkups` and `messages` default to empty lists in the constructor.